test/recipes/65-test_cmp_ctx.t

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env perl
+# Copyright 2007-2019 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright Nokia 2007-2019
+# Copyright Siemens AG 2015-2019
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
```

Comment:
Mixing old and new license form ... should just be one.

Reply:
@t-j-h  We should use  the copyright statement from openssl/CONTRIBUTING, chapter 2. ("All source files should start with ...") in all source files contributed by us, right?

Reply:
Fixed